portal/models/organization.py

```diff
@@ -289,6 +289,35 @@ def rp_from_org(org):
             if rp:
                 return rp
 
+    def invalidation_hook(self):
```

Comment:
Should this be part of some base model class that Organization inherits from?

Reply:
if there was a default implementation, yes.  but in general, we don't need it.

for example, you can use an abstract base class to enforce the subclass implements, but again, the general case is to do nothing, so that would just be empty boiler plate, imo.